statsmodels/tsa/statespace/tests/test_representation.py

```diff
@@ -666,8 +666,8 @@ def test_initialization():
     initial_state = np.zeros(2,) + 1.5
     initial_state_cov = np.eye(2) * 3.
     mod.initialize_known(initial_state, initial_state_cov)
-    assert_equal(mod._initial_state.sum(), 3)
-    assert_equal(mod._initial_state_cov.diagonal().sum(), 6)
+    assert_equal(mod.initialization.constant.sum(), 3)
+    assert_equal(mod.initialization.stationary_cov.diagonal().sum(), 6)
```

Comment:
The attributes being replaced are private.  Is `initialization` intended to be used directly?

Reply:
Yes, these may be created and manipulated by end-users, even separate from model classes (e.g. in principle someone might want to construct their own initialization even for an SARIMAX class, if e.g. they had some kind of prior).